reactioncommerce/reaction
# feat: remove startup template import

Impact: **breaking**  
Type: **feature**

Part of epic #4996

## Issue
There is startup code that loads template assets from `/private`. This feature isn't used in 2.x because only emails currently use templates, and they are loaded from code files by the `email-templates` plugin.

## Solution
Because loading these static assets is a Meteor feature, we've removed this startup code.

## Breaking changes
The following are removed:
- `Reaction.registerTemplate`
- Automatic SimpleSchema validation on `Templates` collection

## Testing
- Verify app starts
- Verify viewing and editing templates on "Settings > Template" page of Reaction Admin works as before.
